tests/comparisons/Controller/testBakeActions.php

```diff
@@ -122,11 +122,10 @@ public function delete($id = null)
     {
         $this->request->allowMethod(['post', 'delete']);
         $bakeArticle = $this->BakeArticles->get($id);
-        if ($this->BakeArticles->delete($bakeArticle)) {
-            $this->Flash->success(__('The bake article has been deleted.'));
-        } else {
+        if (!$this->BakeArticles->delete($bakeArticle)) {
```

Comment:
That is behavior change here. Both messages are now triggered.

Reply:
Ah, you are right.